twint/output.py

```diff
@@ -43,8 +46,9 @@ def _output(obj, output, config):
         else:
             try:
                 print(output)
-            except UnicodeEncodeError:
                 pass
+            except UnicodeEncodeError:
+                print("unicode error")
```

Comment:
Not sure that's necessary

Reply:
Again my changes in output.py were to give clarity where the problems were previously hidden. Main fixes are in user.py